tests/test_examples/test_example_metadata.py

```diff
@@ -8,18 +9,18 @@
 
 
 @pytest.mark.parametrize("file, example", _metadata_examples().items())
-def test_schema_example_filenames(file, example):
+def test_schema_example_filenames(file: str, example: dict) -> None:
```

Comment:
```suggestion
@pytest.mark.parametrize("file", _metadata_examples().keys())
def test_schema_example_filenames(file: str) -> None:
```

`items()` gives you both the dict keys (`file`) and the dict values (`example`). Since this is only testing the keys, I think you can just take only the keys from the dictionary.